tests/unit/tabular/test_validate.py

```diff
@@ -0,0 +1,24 @@
+from typing import Callable
+
+
+def test_validate_simple(specimen: dict, validate: Callable):
+    """Should validate."""
+    result = validate(specimen)
+    assert result.exception is None
+
+
+def test_validate_no_resource_type(specimen: dict, validate: Callable):
+    """Should not validate."""
+    del specimen['resourceType']
+    result = validate(specimen)
+    assert result.exception is not None
+    assert 'resourceType' in str(result.exception)
+
+
+def test_validate_no_bad_date_time(specimen: dict, validate: Callable):
+    """Should not validate."""
+    specimen['receivedTime'] = '1234ABCD'
+    result = validate(specimen)
```

Comment:
Pydantic throws an exception here because of the invalid date and doesn't get to the assert statements. Might need to wrap this in a try/except of some sort

Reply:
All other tabular specific new tests pass

Reply:
Hmm... just tried running this test to confirm and I'm not getting the Pydantic exception.

![Screenshot 2023-12-20 at 12 26 58 PM](https://github.com/ACED-IDP/gen3_util/assets/15309567/67cfe293-0111-4dbd-b47b-002bc4a475e1)

I wonder if this might be due to a difference in test setup (running in visual studio vs command line)?

Reply:
done